build.gradle

```diff
@@ -148,7 +149,7 @@ subprojects {
             }
         }
         dependencies {
-            compileOnly 'com.github.pengrad:jdk9-deps:1.0'
+//            compileOnly 'com.github.pengrad:jdk9-deps:1.0'
```

Comment:
@TranceLove what were this dependency used for? this causes the build failure. the builds pass even without this; can we safely remove it?

Reply:
This was introduced when moving towards JDK 9 per #1941. Perhaps this is no longer needed when officially Android apps can build with JDK 11/17?

Reply:
> This was introduced when moving towards JDK 9 per #1941. Perhaps this is no longer needed when officially Android apps can build with JDK 11/17?

Code complies & the tests pass -- shouldn't it be good? 🤔